<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InterfaceSetup</class>
 <widget class="QWidget" name="InterfaceSetup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>接口过滤</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>方法：</string>
    </property>
   </widget>
   <widget class="QComboBox" name="methodComboBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>51</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>ALL</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>GET</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>POST</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>字段：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="filterLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="allTestCBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>接收全部接口</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>361</width>
     <height>211</height>
    </rect>
   </property>
   <property name="title">
    <string>其他参量监测</string>
   </property>
   <widget class="QCheckBox" name="batteryCBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>电池温度</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuTempCBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU温度</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="batteryLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>90</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cpuTempLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>120</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="batteryLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>警戒值(M)：</string>
    </property>
   </widget>
   <widget class="QLabel" name="cpuTempLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>警戒值(%)：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="wifiCBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Wifi流量</string>
    </property>
   </widget>
   <widget class="QLabel" name="wifiLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>警戒值（M）：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wifiLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="mobileLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>警戒值（M）：</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="mobileCBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mobile流量</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="mobileLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="memCBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>内存</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cpuCBtn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CPU</string>
    </property>
   </widget>
   <widget class="QLabel" name="memLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>警戒值(M)：</string>
    </property>
   </widget>
   <widget class="QLabel" name="cpuLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>警戒值(%)：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="memLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="cpuLineEdit">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>IP &amp;&amp; Port</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IP:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPLineEdit">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>20</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>20</y>
      <width>31</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
